Adds a Security known issue for serverless and 9.0.0 (#1100)
elastic/docs-content: release-notes/elastic-cloud-serverless/known-issues.md

% **Workaround** 
% Workaround description.

:::

> :::{dropdown} In Elastic Security Serverless, installing an {{elastic-defend}} integration or a new agent policy upgrades installed prebuilt rules, reverting user customizations and overwriting user-added actions and exceptions

Reviewer: ```suggestion
:::{dropdown} In {{serverless-full}}, installing an {{elastic-defend}} integration or a new agent policy upgrades installed prebuilt rules, reverting user customizations and overwriting user-added actions and exceptions.
```

> On April 10, 2025, it was discovered that when you install an {{elastic-defend}} integration or a new agent policy for this integration, the installed prebuilt detection rules are upgraded to their latest versions (if any new versions are available). The upgraded rules lose any user-added rule actions, exceptions, and any user customizations. 

Reviewer: ```suggestion
On April 10, 2025, it was discovered that when you install a new {{elastic-defend}} integration or agent policy, the installed prebuilt detection rules upgrade to their latest versions (if any new versions are available). The upgraded rules lose any user-added rule actions, exceptions, and customizations. 
```
